### ISSUE TITLE
bug: Possible incorrect path issue?

### ISSUE THREAD

**Issue by Ashvith10** [specialdoom/proi-ui#90]
I am making use of pnpm (which shouldn't cause any issues, in my opinion). I've tried both CSS as well as JS method, for which I am getting the same issue.

```sh
Failed to load url ./tags/Badge.svelte (resolved id: ./tags/Badge.svelte). Does the file exist?
Error: Failed to load url ./tags/Badge.svelte (resolved id: ./tags/Badge.svelte). Does the file exist?
    at loadAndTransform (file:///home/ashvith/Desktop/portfolio/node_modules/.pnpm/vite@4.0.3/node_modules/vite/dist/node/chunks/dep-0bae2027.js:39278:21)
```

From what I've noticed, looks like `Badge.svelte` is inside `./tags/badge/Badge.svelte` and not `./tags/Badge.svelte`

**Comment by specialdoom** [comment]
I think you are right. I'm going to fix it this evening. Thanks a lot! 😄

**Comment by specialdoom** [comment]
It should be fixed in [v3.7.0](https://github.com/specialdoom/proi-ui/releases/tag/v3.7.0)
